stabilizer/src/main/java/com/hazelcast/stabilizer/worker/tasks/AbstractWorkerTask.java

```diff
@@ -37,7 +38,7 @@
     long iteration = 0;
 
     public AbstractWorkerTask() {
-        this.selector = createOperationSelector();
+        this.selector = createOperationSelectorBuilder().build();
```

Comment:
Ok. Makes sense.

Would it not make sense to pass in the selector builder into the constructor of the worker? This way you don't need to override a method.